### ISSUE TITLE
Remote Configs API causes crashes in HTML5 build

### ISSUE THREAD

**Issue by MaratGilyazov** [GameAnalytics/GA-SDK-DEFOLD#29]
When I call either "**gameanalytics.isRemoteConfigsReady()**" or "**gameanalytics.getRemoteConfigsValueAsString()**" in HTML5 build, it causes a crash with 
```
ERROR:SCRIPT: Unbalanced Lua stack, expected (1), actual (2)
printErr — dmloader.js:347
put_char — BouncyCatapultKing_wasm.js:1:41047
write — BouncyCatapultKing_wasm.js:1:39694
write — BouncyCatapultKing_wasm.js:1:84039
doWritev — BouncyCatapultKing_wasm.js:1:104471
___syscall146 — BouncyCatapultKing_wasm.js:1:128164
wasm-stub
<?>.wasm-function[4775]
<?>.wasm-function[4847]
<?>.wasm-function[4902]
<?>.wasm-function[2917]
<?>.wasm-function[1829]
<?>.wasm-function[534]
<?>.wasm-function[2295]
<?>.wasm-function[2398]
<?>.wasm-function[2297]
<?>.wasm-function[2266]
<?>.wasm-function[5013]
wasm-stub
(anonymous function) — BouncyCatapultKing_wasm.js:1:350935
invoke_vii — BouncyCatapultKing_wasm.js:1:302011
<?>.wasm-function[5013]
<?>.wasm-function[2290]
<?>.wasm-function[2301]
<?>.wasm-function[2265]
<?>.wasm-function[1825]
<?>.wasm-function[1835]
<?>.wasm-function[1974]
<?>.wasm-function[1961]
<?>.wasm-function[1968]
<?>.wasm-function[1815]
<?>.wasm-function[851]
<?>.wasm-function[715]
<?>.wasm-function[598]
<?>.wasm-function[602]
<?>.wasm-function[5012]
wasm-stub
(anonymous function) — BouncyCatapultKing_wasm.js:1:350367
browserIterationFunc — BouncyCatapultKing_wasm.js:1:241043
runIter — BouncyCatapultKing_wasm.js:1:223077
Browser_mainLoop_runner — BouncyCatapultKing_wasm.js:1:242643
print — dmloader.js:346
```

Gameanalytics extension itself seems working: 
```
[Log] INFO:GameAnalytics: Registered gameanalytics Lua extension (dmloader.js, line 346)
[Log] INFO:GameAnalytics: GameAnalytics fully initialized! (dmloader.js, line 346)
```
I'm not calling it from anonimous functions, reproduced even when call from "init" method of the main collection. 

GA 3.1.7, Defold Editor 1.2.167

Can you check, please? 
Thanks.

**Comment by the1schwartz** [comment]
Ok thanks for reporting this. I will look into this.

**Comment by the1schwartz** [comment]
That should be fixed now in the latest version (v3.1.8)
